dimitriosv/week3/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "MyApp",
+  "version": "0.0.1",
+  "description": "",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "babel-core": "^6.5.2",
+    "babel-loader": "^6.2.3",
+    "babel-preset-es2015": "^6.5.0",
+    "babel-preset-react": "^6.5.0",
+    "babel-preset-stage-0": "^6.5.0",
+    "react": "^0.14.7",
+    "react-dom": "^0.14.7",
+    "webpack": "1.12.13"
+  },
+  "dependencies": {
+    "webpack": "~1.12.14"
+  }
```

Comment:
since we are bundling all files (including dependencies) you only need to add your application's dependencies into `devDependencies`. They are not needed in production because they will have been bundled into your application already.

Also, since webpack is used for packaging your application, then I think it should always be in `devDependencies`